modules/assim.sequential/R/Localization.R

```diff
@@ -12,9 +12,10 @@
 
 Local.support <-function(Pf, distance.mat, scalef=1){
   #making a matrix as the size of the Pf
-  rho <- exp((-1*distance.mat^2)/(2*scalef^2))
-  
-  return(Pf*rho)
+  rho <- exp(((-1) * distance.mat^2) / (2 * scalef^2))
+  # zeroing out values that are really small
+  rho <- round(rho, 3)
```

Comment:
This is rounding off all values, not just zeroing out values < 0.001. If the latter is the goal (as stated) why not do that explicitly?